integration-tests/support/pageObjects/createApplication-po.ts

```diff
@@ -1,6 +1,12 @@
 export const addComponentPagePO = {
   samples: 'Start with a sample.',
   addComponent: '[data-testid="add-component"]',
+  enterSource: '[id="form-input-source-field"]',
+  validated: '[id="form-input-source-field-helper"]',
+  gitOptions: 'Git options',
+  gitReference: '[id="form-input-git-reference-field"]',
+  contextDir: '[id="form-input-git-contextDir-field"]',
+  next: '[data-test="submit-button"]'
```

Comment:
Shouldn't we be consistent here. Some have `data-testid`, some have `data-test` and some `id`?

Reply:
Sure, I've made some changes in the corresponding files of the components by adding a new `data-testid` field as it wasn't present before, and then updated the test accordingly.

Reply:
@jrichter1 Do you remember what is the standard (`data-test` or `data-testid`) we use across console?

Reply:
there is the legacy `data-test-id` and the current  is `data-test`

Reply:
so should I replace `data-testid` with `data-test`?